lib/config.js

```diff
@@ -221,6 +221,20 @@ const conf = convict({
       env: 'WORKER_URL'
     }
   },
+  authServerMessaging: {
+    region: {
+      doc: 'The region where the queues live',
+      format: String,
+      env: 'AUTH_SERVER_MESSAGING_REGION',
+      default: ''
+    },
+    profileUpdatesQueueUrl: {
+      doc: 'The bounce queue URL to use (should include https://sqs.<region>.amazonaws.com/<account-id>/<queue-name>)',
+      format: String,
+      env: 'PROFILE_UPDATES_QUEUE_URL',
+      default: ''
+    }
+  },
```

Comment:
Let's get @jbuck and @jrgm input on this as well; guys, thoughts on using an SQS queue here to relay notifications back to the auth-server, and from there on to the connected devices?